sdk/communication/communication-administration/samples/javascript/releasePhoneNumbers.js

```diff
@@ -22,44 +22,16 @@ async function main() {
   // create an instance of PhoneNumberAdministrationClient
   const phoneNumberClient = new PhoneNumberAdministrationClient(connectionString);
 
-  console.log("Getting acquired phone numbers.");
+  // Release a phone numbers
+  const releasePoller = await phoneNumberClient.beginReleasePhoneNumbers([
+    "+14125550000",
+    "+14125550001"
+  ]);
```

Comment:
How is this going to work repeatedly? From what I remember, a phone number cannot be immediately acquired once it has been released.

Reply:
This is a sample so it's not meant to work repeatedly. Also the numbers being released are fake (555). I will update the phone numbers though. I just read that only 555-0100 to 555-0199 are reserved for fake numbers.